packages/http-specs/package.json

```diff
@@ -9,7 +9,7 @@
     "build": "tsc -p ./tsconfig.build.json",
     "clean": "rimraf dist/ temp/",
     "test:e2e": "pnpm validate-scenarios  && pnpm validate-mock-apis && pnpm validate-client-server",
-    "validate:all": "pnpm build && pnpm regen-docs && pnpm test:e2e",
+    "validate:all": "prettier specs --write && pnpm build && pnpm regen-docs && pnpm test:e2e",
```

Comment:
Not sure if you want to add `--write` here, as this is `validate`, not `fix`

Reply:
According to experience, it will be much convenient for developers to validate the scenarios instead of running a series of command before creating a PR.

Reply:
Sure, if e.g. `regen-docs` already modifies the files, it likely won't matter to have one more command that write files.

Reply:
`regen-docs` won't format the files which may cause format check failure in CI if devs forget to run format command before commit. So I try to add all necessary command into `validate:all` so that devs could get one command for all necessary checks/fix before commit.

Reply:
updated.

Reply:
please revert this and add to the prettierignore we shouldn't format generated files